src/app/services/case/case.service.ts

```diff
@@ -37,6 +39,23 @@ export class CaseService {
     );
   }
 
+  // Single get case file
+  getCaseFile(caseId: string | number): Observable<CaseFile> {
+    const apiURL = `${GET_CASE_PATH}/${caseId}`;
+    return this.http.get<CaseFile>(apiURL).pipe();
+  }
+
+  // Single get case hearings
+  getCaseHearings(caseId: number): Observable<HearingData[]> {
+    const apiURL = `${GET_CASE_PATH}/${caseId}/hearings`;
+    return this.http.get<HearingData[]>(apiURL).pipe(
+      catchError((err: Error) => {
+        this.errorHandlerService.handleError(err);
```

Comment:
We can probably remove these catchError blocks / errorHandlerService calls from the service layer as it's being caught, logged and re thrown by global HTTP error interceptor.